math/scalapack/Portfile

```diff
@@ -28,8 +28,20 @@ github.tarball_from archive
 
 mpi.setup           require require_fortran
 
+# fix wrong version, which results in wrong versioned name of libscalapack.${version}.dylib
+post-patch {
+    reinplace "s|2.2.1|2.2.2|g" ${worksrcpath}/CMakeLists.txt
+}
+
 compiler.blacklist-append   {clang < 500}
 
+# remove duplicate rpath ${prefix}/lib/libgcc
+compilers.add_gcc_rpath_support no
+# remove duplicate rpath ${prefix}/lib
+post-build {
+    system -W ${workpath}/build/lib/ "install_name_tool -delete_rpath ${prefix}/lib libscalapack.${version}.dylib"
```

Comment:
`-delete_rpath` is not supported in 10.5 and below, so should be used conditionally.

Reply:
ok. done so. I have no idea, whether the issue is relevant for 10.5 and below. If so, no idea how to resolve it.